src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/cft/query/SelectTaskResourceQueryBuilder.java

```diff
@@ -8,18 +8,27 @@
 
 public class SelectTaskResourceQueryBuilder extends TaskResourceQueryBuilder<TaskResource> {
 
+    private boolean taskRoles;
```

Comment:
maybe rename the var to `shouldFetchTaskRoles` or something similar?

Reply:
Done